wpilibj/src/athena/java/edu/wpi/first/wpilibj/AnalogInput.java

```diff
@@ -258,21 +242,21 @@ public void resetAccumulator() {
   /**
    * Set the center value of the accumulator.
    *
-   * The center value is subtracted from each A/D value before it is added to
-   * the accumulator. This is used for the center value of devices like gyros
-   * and accelerometers to take the device offset into account when integrating.
+   * <p>The center value is subtracted from each A/D value before it is added to the accumulator.
+   * This is used for the center value of devices like gyros and accelerometers to take the device
+   * offset into account when integrating.
```

Comment:
Comments are being formatted to longer than 80 characters, which is a bit ugly in editors like Vim. Are we OK with this?